cmds/rm/rm_test.go

```diff
@@ -71,7 +71,7 @@ func Test_rm_1(t *testing.T) {
 	fmt.Println("== Deleting files and empty folders (no args) ...")
 	files := []string{path.Join(d, "hi1.txt"), path.Join(d, "hi2.txt"), path.Join(d, "go.txt")}
 
-	flags.v = true
+	*verbose = true
```

Comment:
Changing this here changes it for everything. It'd be nicer if you passed this as an argument to `rm`, because that means this doesn't have a side effect on any subsequent tests.

(Actually, go tests can be run in parallel, so setting it here could affect another thread where you expect rm not to be verbose.)